submissions/charlesBanda/html/html-forms/Contact.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
This file is not necessary. You only need one contact page. And the contact page should be inside the `html-forms` folder. Delete this one and leave the other one